### ISSUE TITLE
Preserve comments in the produced .env.example file.

### ISSUE THREAD

**Issue by dashmug** [hija/clean-dotenv#3]
It would be great if this tool can preserve comments.

For example, with this example `.env`...

```env
# Copy and paste the credentials here.
#
# Restart running containers whenever these values are updated.

AWS_PROFILE="default"

# Alternatively, if you wish to use keys directly, uncomment the 
# following lines and provide the values. Comment out or remove the 
# AWS_PROFILE line above.

# AWS_ACCESS_KEY_ID=""
# AWS_SECRET_ACCESS_KEY=""
# AWS_SESSION_TOKEN=""
```

I would expect to get this as the `.env.example`.

```env
# Copy and paste the credentials here.
#
# Restart running containers whenever these values are updated.

AWS_PROFILE=""

# Alternatively, if you wish to use keys directly, uncomment the 
# following lines and provide the values. Comment out or remove the 
# AWS_PROFILE line above.

# AWS_ACCESS_KEY_ID=""
# AWS_SECRET_ACCESS_KEY=""
# AWS_SESSION_TOKEN=""
```

Currently, the tool produces this...

```env
AWS_PROFILE=
```

**Comment by hija** [comment]
Hi @dashmug,

thank you for trying out the pre-commit hook and for giving feedback!

I really like your idea and think this should definitely be implemented. Unfortunately, from a quick glimpse it does not seem like the .env file parser we are using currently is (out of the box) suitable for this task. I think we need to choose another .env parser or write our own one.

I will add this feature to the feature request list, but since I am a bit busy, I do not think I am able to implement it in the near future. However, feel free to contribute to the package :)

**Comment by hija** [comment]
Implemented in #4. 🚀 

Feel free to open a new issue in case anything does not work as expected.
